docs/modules/ROOT/pages/node_pattern.adoc

```diff
@@ -409,6 +409,43 @@ has_user_data?(node, /^pass(word)?$/i)
 has_user_data?(node, ->(key) { # return true or false depending on key })
 ----
 
+== `%keyword` for named parameters
+
+Arguments can be passed as named parameters. They will be matched using `===`.
+Defaults values can be passed to `def_node_matcher` and `def_node_search`.
+
+[source,ruby]
+----
+def_node_matcher :interesting_call?, '(send _ %method ...)',
+                 method: Set[:transform_values, :transform_keys,
+                             :transform_values!, :transform_keys!,
+                             :to_h].freeze
+
+# Usage:
+
+interesting_call?(node) # use the default methods
+interesting_call?(node, method: /^transform/) # match anything starting with 'transform'
+----
+
+Named parameters as arguments to custom methods are not supported.
```

Comment:
You can put this in a "NOTE:" so it would stand out.

Reply:
Sure. I didn't support them because I didn't see the need, but now that I think about it, it's trivial to support so probably I should.

Reply:
I added support. Implementation took about 2 minutes, +5 for tests +5 for a refactor to make RuboCop happy +10 to open a [bug report](https://github.com/rubocop-hq/rubocop/issues/8158) 😆